tasks/sysctl-setup.yml

```diff
@@ -7,7 +7,24 @@
     ansible_distribution != 'Debian'
     or ansible_distribution_major_version | int < 10
 
-# See: https://kubernetes.io/docs/setup/production-environment/tools/kubeadm/install-kubeadm/#letting-iptables-see-bridged-traffic
+# See: https://kubernetes.io/docs/setup/production-environment/container-runtimes/#forwarding-ipv4-and-letting-iptables-see-bridged-traffic
+- name: Ensure relevant kernel module is enabled
+  modprobe:
+    name: br_netfilter
+    state: present
+  loop:
+    - br_netfilter
+    - overlay
+
+- name: Persist kernel modules after reboot
+  lineinfile:
```

Comment:
By using lineinfile, I circumvent situations where a system may be configured with more than just these lines. Perhaps a separate file is more acceptable?